airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "תצוגת גרף כברירת מחדל",
+  "defaultToGridView": "תצוגת רשת כברירת מחדל",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
+    "assets": "נכסים (Assets)",
+    "browse": "עיין",
+    "admin": "מנהל",
+    "docs": "תיעוד",
+    "plugins": "פלאגינים"
+  },
+  "browse": {
+    "auditLog": "לוג מערכת",
+    "xcoms": "(XComs)"
+  },
+  "admin": {
+    "Variables": "משתנים",
+    "Pools": "בריכת משאבים",
+    "Providers": "חבילות עזר",
+    "Plugins": "פלאגינים",
```

Comment:
```suggestion
    "Plugins": "תוספים",
```

Reply:
Please notice that you approved `תוספים` before (in the `nav`), but left this one as-is